### ISSUE TITLE
parameter or options changes should auto update or at least clear plots and table

### ISSUE THREAD

**Issue by thcasey3** [ylin00/odnplab#12]
any change to the parameters or options should re-run calculations automatically or at least clear the plots and table so users can't forget to re-calculate when they make a change. I like how "click me" comes up but it might not be obvious enough

**Comment by ylin00** [comment]
Agree. Ideally we want the results to be updated as soon as the parameters changed. However, the calculation takes ~10seconds so this will not be possible until the performance gets improved.

I chose not to erase the results based on my following imaginations.  

As a user analyzing an experiment, I want to be able to run an analysis, check the results, adjust parameters and re-analysis -- an iteration.  And I want this iteration to be fast. 1-2 seconds might work, 10 seconds is too long if I have to iterate several times. This means I prefer iterate 2-3 times and for each iteration, make multiple adjustments to the parameters while looking at the results from the previous analysis. 

Again. This is just my guess. We will make the change and give it a try to a few other users. I will keep this issue open for more discussion.

**Comment by thcasey3** [comment]
You should be able to just have the processing run once when the data are loaded and the fit/calculations repeat with any adjustments, the processing is what takes time, the calculations are very fast. For a web app do you have something like the "self" structure to keep variables around?
